packages/@aws-cdk/cloudformation-include/test/yaml-templates.test.ts

```diff
@@ -0,0 +1,397 @@
+            "CidrBlock": {
+              "Fn::Cidr": [
+                "192.168.1.1/24",
+                2,
+                5,
+              ],
+            },
+          },
+        },
+        "CidrVpc2": {
+          "Type": "AWS::EC2::VPC",
+          "Properties": {
+            "CidrBlock": {
+              "Fn::Cidr": [
+                "192.168.1.1/24",
+                "2",
+                "5",
+              ],
+            },
+          },
+        },
+      },
+    });
+  });
+
+  test('can ingest a template with the short form !FindInMap', () => {
+    includeTestTemplate(stack, 'short-find-in-map.yaml');
+
+    expect(stack).toMatchTemplate({
+      "Mappings": {
+        "RegionMap": {
+          "region-1": {
+            "HVM64": "name1",
+            "HVMG2": "name2",
+          },
+        },
+      },
+      "Resources": {
+        "Bucket": {
+          "Type": "AWS::S3::Bucket",
+          "Properties": {
+            "BucketName": {
+              "Fn::FindInMap": [
+                "RegionMap",
+                "region-1",
+                "HVM64",
+              ],
+            },
+          },
+        },
+      },
+    });
+  });
+
+  test('can ingest a template with the short form !GetAtt', () => {
+    includeTestTemplate(stack, 'short-get-att.yaml');
+
+    expect(stack).toMatchTemplate({
+      "Resources": {
+        "Bucket1": {
+          "Type": "AWS::S3::Bucket",
+          "Properties": {
+            "BucketName": "SomeBucketName",
+            "ReplicationConfiguration": {
+              "Role": "some-role",
+              "Rules": [
+                {
+                  "Destination": {
+                    "Bucket": "Bucket1",
+                    "StorageClass": "STANDARD",
+                  },
+                  "Status": "Enabled",
+                },
+              ],
+            },
+          },
+        },
+        "Bucket2": {
+          "Type": "AWS::S3::Bucket",
+          "Properties": {
+            "BucketName": {
+              "Fn::GetAtt": [
+                "Bucket1",
+                "BucketName",
+              ],
+            },
+          },
+        },
+        "Bucket3": {
+          "Type": "AWS::S3::Bucket",
+          "Properties": {
+            "BucketName": {
+              "Fn::GetAtt": [
+                "Bucket1",
+                "BucketName",
+              ],
+            },
+          },
+        },
+        "Bucket4": {
+          "Type": "AWS::S3::Bucket",
+          "Properties": {
+            "BucketName": {
+              "Fn::GetAtt": [
+                "Bucket1",
+                "Rules.Destination",
+              ],
+            },
+          },
+        },
+        "Bucket5": {
+          "Type": "AWS::S3::Bucket",
+          "Properties": {
+            "BucketName": {
+              "Fn::GetAtt": [
+                "Bucket1",
+                "Rules.Destination",
+              ],
+            },
+          },
+        },
+      },
+    });
+  });
```

Comment:
This assertion is way too long. You had me before, but here you lost me. This template needs to be much, much shortened.

Also, be mindful of your reader. Instead of

```json
        "Bucket5": {
          "Type": "AWS::S3::Bucket",
          "Properties": {
            "BucketName": {
              "Fn::GetAtt": [
                "Bucket1",
                "Rules.Destination",
              ],
            },
          },
        },
```

Make their life easier, and do:

```json
        "Bucket5": {
          "Type": "AWS::S3::Bucket",
          "Properties": {
            "BucketName": { "Fn::GetAtt": ["Bucket1", "Rules.Destination"] },
          },
        },
```

This goes for all assertions.

Remember, that reader that you're polite towards today will be you in one month, fixing a bug in here!